js/mainjs/initWindow.js

```diff
@@ -49,13 +48,12 @@ module.exports = function(config) {
 
 	// Load the index.html of the app.
 	mainWindow.loadURL('file://' + __dirname + '/../../index.html');
-
 	// Choose not to show the menubar
 	if (process.platform !== 'darwin') {
 		mainWindow.setMenuBarVisibility(false);
 	} else {
 		// Create the Application's main menu - OSX version might feel weird without a menubar
-		Menu.setApplicationMenu(AppMenu);
+		Menu.setApplicationMenu(new AppMenu(mainWindow));
```

Comment:
I amended the commits to reflect their issue fixes.  I initially called this as Menu.setApplicationMenu(AppMenu()), and JSHint yelled at me for not calling a constructor using new.  Looking into it more, it looks like jshint expects all functions that start with caps to be constructors, invoked using new.  AppMenu() can still be called as a function, but should probably be renamed to reflect this convention